README.md

```diff
@@ -7,7 +7,8 @@ Appenders for .NET logging frameworks which send data to Sumo Logic HTTP sources
 | In keeping with industry standard security best practices, as of May 31, 2018, the Sumo Logic service will only support TLS version 1.2 going forward. Verify that all connections to Sumo Logic endpoints are made from software that supports TLS 1.2. |
 
 # Prerequisites
-* .NET 4.5 or later or .NET Standard 1.5
+* .NET 4.5 or later or .NET Standard 1.5 (for NLog and Log4Net)
+* .NET 4.6.1 or later or .NET Standard 2.0 (for Serilog)
```

Comment:
Why .NET 4.6.1 is required here? I'm asking since the targetFramework in .csproj is 4.5 as well as the minimum requirement in `SumoLogic.Logging.Serilog/README.txt`
It's better to still keep .NET 4.5 if possible since we don't want asking all users upgrade their .NET to adopt this change

Reply:
@bin3377 In my original project I was simply following guidance from [Cross-platform targeting](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting).

Since I was _adding_ new project, I kept it like that. Now I lowered it to .NET 4.5 so current users of SumoLogic won't need to do upgrade when using SeriLog. (... despite I believe those people *should* update)